lib/sso_openid/urls.yml

```diff
@@ -1,4 +1,7 @@
 defaults:
+  auctions:
+    host: "auctions.networkforgood-beta.com"
+    fqdn: "https://auctions.networkforgood-beta.com"
```

Comment:
I think this should be fqdn: "https://api.auctions.networkforgood-beta.com". In order for a request to get to the new auctions applications, it needs a subdomain, then "auctions", then the nfg domain. 

This endpoint doesn't exist yet on auctions, but it is ok to include it here for now